docs/templates/snakemake/README.md

```diff
@@ -0,0 +1,59 @@
+# {Snakemake pipeline template}
+
+## How to create your snakemake pipeline for APAeval
+Read this section, but do NOT include it your final README.
+> * Copy the whole directory `snakemake` with its' subfolders to suitable location.
+> * Rename `snakemake` to `[METHOD]_snakemake`, [METHOD] being the name of the tool you're building a workflow for.
+> * Adjust names and contents of files that appear in this [NOTATION].
+> * Place any scripts or subworkflows you're going to use into the directories `workflow/scripts` and `workflow/rules`, respectively.
+> * Write your workflow in `Snakefile`.
+> * Here are the [snakemake docs](https://snakemake.readthedocs.io/en/stable/index.html)
+> * General good practices:
+>     * Test your code with [snakemake --lint](https://snakemake.readthedocs.io/en/stable/snakefiles/writing_snakefiles.html#best-practices).
+>     * One (shell) command per rule. 
+>     * If samples differ in a meaningful way (e.g. single end and paired end samples), it might be better to write subworkflows within `workflow/rules`. 
+>     * If applicable, each rule has it's own conda environment. 
+> * There are some shell scripts which can be used to start a snakemake run. Adjust the name of the `config file` in these scripts:
+>     * `dryrun.sh`
+>     * `rulegraph.sh` (Also adjust name of output `.png`)
+>     * `run_local.sh`
+> * Adjust this `README.md`: Delete this description section and populate the sections below with your awesome experience ;)
+
+# [METHOD]
+
+{Description of method, with link to publication and source code.}
```

Comment:
I don't see why they should spend time filling this out, as it is already in the methods table. I would instead put here a link to the methods table and ask the people to specify the name of the method tested EXACTLY AS IT IS LISTED in the Algorithm column of that table

Reply:
Added a comment that method name should be the same as in the spreadsheet. Did not hardlink to the google spreadsheet as I'm not sure of the write accesses.